### PR TITLE
Green power 3 support

### PR DESCRIPTION
**Pull request by Gautomax**

### CODE REVIEW

**Comment by Gautomax** [comment]
Add GreenPower_3 (Legrand Wireless and batteryless bind control switch) support

File: src/devices/legrand.js [hunk 1]
```diff
@@ -447,6 +447,30 @@ module.exports = [
         toZigbee: [],
         exposes: [e.action(['press_once', 'press_twice'])],
     },
+    {
+        zigbeeModel: ['GreenPower_3'],
```
Comment (Koenkk) [inline src/devices/legrand.js:451]:
```suggestion

```

Comment (Koenkk) [reply src/devices/legrand.js:451]: Please remove this line, not needed because of the `fingerprint`

File: src/devices/legrand.js [hunk 2]
```diff
@@ -447,6 +447,30 @@ module.exports = [
...
+        fingerprint: [{modelID: 'GreenPower_3', ieeeAddr: /^0x00000000005.....$/}],
+        model: '600087L',
+        vendor: 'Legrand',
+        description: 'Wireless and batteryless blind control switch',
+        fromZigbee: [{
+            cluster: 'greenPower',
+            type: ['commandNotification'],
+            convert: (model, msg, publish, options, meta) => {
```
Comment (Koenkk) [inline src/devices/legrand.js:459]: Please move this converter to an `fzLocal` in the top of this file (https://github.com/Koenkk/zigbee-herdsman-converters/blob/3080896a20a743b6bd79287e9fc7f7418e2b3046/src/devices/legrand.js#L29)

Comment (Gautomax) [reply src/devices/legrand.js:458]: Ok

Comment (Gautomax) [reply src/devices/legrand.js]: Done

**Comment by Gautomax** [comment]
New pull request with updates

**Pull request closed without merging**